Bottom: 2ce544984cec31aa168f50bc1e7e8c8263785118
Top:    5c5cfcad27124334c1abd2e2d4c4ed62d642ab44
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-24 17:00:53 +0100

Refresh of hid-gtk-remove-unused-function

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 58136b6..7686106 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -507,10 +507,9 @@ void ghid_draw_rect (hidGC gc, int x1, int y1, int x2, int y2);
 void ghid_fill_circle (hidGC gc, int cx, int cy, int radius);
 void ghid_fill_polygon (hidGC gc, int n_coords, int *x, int *y);
 void ghid_fill_rect (hidGC gc, int x1, int y1, int x2, int y2);
-void ghid_invalidate_lr ();
-void ghid_invalidate_all ();
 
 /* gtkhid-main.c */
+void ghid_invalidate_all ();
 void ghid_get_coords (const char *msg, int *x, int *y);
 gint PCBChanged (int argc, char **argv, int x, int y);
